From 8e3fdc5e2107823bd8ea2916b7b198bddb734230 Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Thu, 13 Aug 2020 16:26:40 +0700
Subject: [PATCH 359/441] ARM: dts: r8a7744: Remove unused VSP nodes

VSP modules vsp@fe928000, vsp@fe930000, and vsp@fe938000 are used as
VSP for DU and already defined in other nodes (vspm, vspd0, and vspd1).
This patch removes these modules to avoid confliction.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7744.dtsi | 42 ------------------------------------------
 1 file changed, 42 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7744.dtsi b/arch/arm/boot/dts/r8a7744.dtsi
index 9c1abcc..225ee46 100644
--- a/arch/arm/boot/dts/r8a7744.dtsi
+++ b/arch/arm/boot/dts/r8a7744.dtsi
@@ -1585,48 +1585,6 @@
 			power-domains = <&cpg_clocks>;
 		};
 
-		vsp@fe928000 {
-			compatible = "renesas,vsp1";
-			reg = <0 0xfe928000 0 0x8000>;
-			interrupts = <GIC_SPI 267 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&mstp1_clks R8A7744_CLK_VSP1_S>;
-			power-domains = <&cpg_clocks>;
-
-			renesas,has-lut;
-			renesas,has-sru;
-			renesas,#rpf = <5>;
-			renesas,#uds = <1>;
-			renesas,#wpf = <4>;
-		};
-
-		vsp@fe930000 {
-			compatible = "renesas,vsp1";
-			reg = <0 0xfe930000 0 0x8000>;
-			interrupts = <GIC_SPI 246 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&mstp1_clks R8A7744_CLK_VSP1_DU0>;
-			power-domains = <&cpg_clocks>;
-
-			renesas,has-lif;
-			renesas,has-lut;
-			renesas,#rpf = <4>;
-			renesas,#uds = <1>;
-			renesas,#wpf = <1>;
-		};
-
-		vsp@fe938000 {
-			compatible = "renesas,vsp1";
-			reg = <0 0xfe938000 0 0x8000>;
-			interrupts = <GIC_SPI 247 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&mstp1_clks R8A7744_CLK_VSP1_DU1>;
-			power-domains = <&cpg_clocks>;
-
-			renesas,has-lif;
-			renesas,has-lut;
-			renesas,#rpf = <4>;
-			renesas,#uds = <1>;
-			renesas,#wpf = <1>;
-		};
-
 		pciec: pcie@fe000000 {
 			compatible = "renesas,pcie-r8a7744",
 				     "renesas,pcie-rcar-gen2";
-- 
2.7.4

